radixdlt/src/main/java/com/radixdlt/consensus/EpochManager.java

```diff
@@ -166,28 +183,54 @@ public void processEpochChange(EpochChange epochChange) {
 		queuedEvents.remove(nextEpoch);
 	}
 
+	private void processEndOfEpoch(VertexMetadata vertexMetadata) {
+		log.info("{}: END_OF_EPOCH: {}", this.loggerPrefix, vertexMetadata);
```

Comment:
This should be at debug or trace level, as it's in response to an external request and we don't have control over the rate.